charts/kargo/templates/external-webhooks-server/deployment.yaml

```diff
@@ -86,9 +86,11 @@ spec:
           name: webhooks-server
           protocol: TCP
         volumeMounts:
+        {{- if .Values.externalWebhooksServer.tls.enabled }}
         - mountPath: /tmp/k8s-webhook-server/serving-certs
           name: cert
           readOnly: true
+        {{- end }}
```

Comment:
I agree with this change, but now that all mounts are conditional, including `volumeMounts:` probably also needs to be conditional.